LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -258,32 +269,27 @@ public void SplitTenderExample_LevelUp_Then_Cash_With_Adjusted_Exemption_And_Adj
                 $"Expected: {expectedCompletedOrderValues}" + Environment.NewLine +
                 $"Actual: {completedOrderValues}");
         }
-        
+
         /// <summary>
-        /// In this example, cash is applied first. The remaining balance can be covered in full by the LevelUp 
-        /// discount credit available.
+        /// This was an example where more than half of the check was discounted. This was added after a bug
+        /// was found which could result in the tax_amount being incorrectly adjusted (increasing) on the 
+        /// CalculateCompleteOrderValues(); call.
         /// </summary>
-        /// <remarks>
-        /// In this example, the second payment will still be responsible for some tax.
-        /// </remarks>
         [TestMethod]
-        public void SplitTenderExample_LevelUp_Second_Discount_Remaining_Balance()
+        public void SplitTenderExample_NoChangeInSpendOrTax_WhenMostOfCheckIsDiscounted()
         {
             // $22.00 is owed, $2.00 of that is tax. $0.00 of that is tobacco/alcohol, and the customer wants to pay
             // $12.00 towards the check all of which is discountable.
-            Check check = new Check(total: 2200, tax: 200);
+            Check check = new Check(total: 1060, tax: 60);
             int exemptionAmount = 0;
-            int spendAmount = 1200;
-
-            // Apply a non-LevelUp tender
-            check.ApplyTender(1000);
+            int spendAmount = 530;
 
             // Create proposed order: expected values
             AdjustedCheckValues expectedProposedOrderValues =
                 new AdjustedCheckValues(
-                    spendAmount: 1200,
-                    taxAmount: 200,
-                    exemptionAmount: 0);
+                    spendAmount: 530,       // amount the customer wants to pay
+                    taxAmount: 0,           // no tax will be due for this payment; tax will be paid "at the end"
+                    exemptionAmount: 0);    // there are no exemptions
```

Comment:
Checking for clarity and understanding: with the original implementation (prior to this PR) the expected values for proposed order would be the same?